.circleci/config.yml

```diff
@@ -0,0 +1,104 @@
+orbs:
+  slack: circleci/slack@3.4.1
+version: 2.1
+jobs:
+  test:
+    machine:
+      image: ubuntu-1604:201903-01
+    steps:
+      - checkout
+      - run: docker run --name postgres -p 5432:5432 -d postgres:10.6
+      - run:
+          name: Install Node
+          command: |
+            curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+            sudo apt-get install -y nodejs
+            echo 'export PATH=/usr/bin:$PATH' >> $BASH_ENV
+            which node && node -v
+      - run: npm install
+      - run: mkdir -p ~/reports/jest
+      - run:
+          name: Run tests
+          command: JEST_JUNIT_OUTPUT_DIR=$HOME/reports/jest npm test -- --reporters=default --reporters=jest-junit
```

Comment:
I didn't configure tests with Jest IIRC

Reply:
removed it for now, let's see if we can set it up later